Runtime/Scripts/GltfImport.cs

```diff
@@ -1404,7 +1406,9 @@ enum ChunkFormat : uint
         /// </summary>
         /// <returns>Array containing each node's parent node index (or -1 for root nodes)</returns>
         int[] CreateUniqueNames() {
+#if UNITY_ANIMATION
```

Comment:
If I understand this correctly, you intended to disable unique names if animation is not enabled.

1. Why do you want to disable them? I think it's still a desirable feature and glTFast should adhere to wht it offers in `ImportSettings.nodeNameMethod` to the users.
2. Even if there's a good reason, we should probably refactor this a bit so that the method name and description match what it does (in any case)

Reply:
I'm sorry, if I don't understand each part of the code fully, but the problem was that the `nodeNames` variable was in preprocessor-brackets before:

(line 175):
```
#if UNITY_ANIMATION
        /// <summary>
        /// Unity's animation system addresses target GameObjects by hierarchical name.
        /// To make sure names are consistent and have no conflicts they are precalculated
        /// and stored in this array.
        /// </summary>
        string[] nodeNames;
#endif
```
The code wa snot compiling because the variable definition was only defined when `UNITY_ANIMATION` is on, but then used when without the preprocessor makro.
The comment above it made me think it is only needed for animations.

Reply:
As a counter-proposel I have removed all `UNITY_ANIMATION` around the node-names.